docs/application/native/guides/connectivity/softap.md

```diff
@@ -0,0 +1,342 @@
+
+   Define `connected_client_cb()`:
+      ```
+      static bool connected_client_cb(softap_client_h client, void *data)
+      {
+          softap_client_h clone = NULL;
+          char *ip_address = NULL;
+          char *mac_address = NULL;
+          char *hostname = NULL;
+          time_t timestamp;
+          struct tm t;
+
+          if (softap_client_clone(&clone, client) != SOFTAP_ERROR_NONE) {
+              printf("softap_client_clone is failed\n");
+              return false;
+          }
+
+          /* Get information */
+          if (softap_client_get_ip_address(clone, SOFTAP_ADDRESS_FAMILY_IPV4, &ip_address) != SOFTAP_ERROR_NONE)
+              printf("Failed to get client IP address");
+
+          if (softap_client_get_mac_address(clone, &mac_address) != SOFTAP_ERROR_NONE)
+              printf("Failed to get client MAC address");
+
+          if (softap_client_get_name(clone, &hostname) != SOFTAP_ERROR_NONE)
+              printf("Failed to get client name");
+
+          if (softap_client_get_time(clone, &timestamp) != SOFTAP_ERROR_NONE)
+              printf("Failed to get connected time");
+
+          localtime_r(&timestamp, &t);
+
+          printf("\n< Client Info. >\n");
+          printf("\tIP Address %s\n", ip_address);
+          printf("\tMAC Address : %s\n", mac_address);
+          printf("\tHostname : %s\n", hostname);
+          printf("\tTime stamp : %04d-%02d-%02d %02d:%02d:%02d",
+                  t.tm_year + 1900, t.tm_mon + 1,
+                  t.tm_mday, t.tm_hour,
+                  t.tm_min, t.tm_sec);
+
+          if (ip_address)
+              free(ip_address);
+          if (mac_address)
+              free(mac_address);
+          if (hostname)
+              free(hostname);
+
+          softap_client_destroy(clone);
+
+          /* Continue iteration */
+          return true;
+      }
+      ```
+
+
+## Disabling SoftAP
+
+To disable SoftAP:
+
+   ```
+   ret = softap_disable(softap);
+   if (ret != SOFTAP_ERROR_NONE)
+       printf("Failed to disable SoftAP");   
+   ```
+
+## Releasing all resources
+
+To release the SoftAP resources:
+
+1. Unregister callback functions:
+    ```
+    ret = softap_unset_enabled_cb(softap);
+    if (ret != SOFTAP_ERROR_NONE)
+       printf("Failed to unset the callback");
+
+   ret = softap_unset_disabled_cb(softap);
+   if (ret != SOFTAP_ERROR_NONE)
+       printf("Failed to unset the callback");
+
+   ret = softap_unset_client_connection_state_changed_cb(softap);
+   if (ret != SOFTAP_ERROR_NONE)
+       printf("Failed to unset the callback");
+
+   ```
+
+
+2. Destroy SoftAP handle
+
+   When no longer needed, destroy the handle:
+
+   ```
+   ret = softap_destroy(softap);
+   if (ret != SOFTAP_ERROR_NONE)
+       printf("Failed to destory the handle");
+   ```
+
+
+## Related Information
+   - Dependencies
+   - Tizen 5.0 and Higher for Mobile.
```

Comment:
To maintain consistency, please remove the period. 
I have checked other Tizen files.
I apologize for suggesting this before.